projects/igniteui-angular/src/lib/grids/grid/grid.pipes.ts

```diff
@@ -132,7 +132,7 @@ export class IgxGridFilteringPipe implements PipeTransform {
 
     public transform(collection: any[], expressionsTree: IFilteringExpressionsTree,
         filterStrategy: IFilteringStrategy,
-        advancedExpressionsTree: IFilteringExpressionsTree, id: string, pipeTrigger: number) {
+        advancedExpressionsTree: IFilteringExpressionsTree, id: string, pipeTrigger: number, filteringPipeTrigger: number) {
```

Comment:
You don't need to declare the pipetrigger in the parameter list.

Reply:
If I don't declare them the build fails - [https://dev.azure.com/IgniteUI/igniteui-angular/_build/results?buildId=17280&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=b24cf96b-e476-5cd6-97ae-776d6935d22f](url)

Reply:
OK, nvm...